package.json

```
@@ -83,7 +73,8 @@
     "fs-extra": "^10.0.0",
     "match-all": "^1.2.6",
     "murmur-128": "^0.2.1",
-    "qs": "^6.9.4"
+    "qs": "^6.9.4",
+    "zksync-web3": "^0.3.4"
```

Please note that this version is outdated, there was a breaking change in the server that removes `valid_from`/`valid_until` fields from tx.
```suggestion
    "zksync-web3": "^0.4.0"
```